Portfolio.html

```diff
@@ -4,11 +4,85 @@
     <meta charset="UTF-8">
     <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="Portfolio.css">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="Media-Query.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Rob Reeves Portfolio</title>
 </head>
-<body>
-
```

Comment:
The reason this is red is because you are missing a body tag - see the comment in line 67 - somehow your body tag has slipped down here.

Reply:
done.